src/main/java/seedu/address/MainApp.java

```diff
@@ -140,18 +147,24 @@ protected Config initConfig(Path configFilePath) {
      */
     protected UserPrefs initPrefs(UserPrefsStorage storage) {
         Path prefsFilePath = storage.getUserPrefsFilePath();
-        logger.info("Using prefs file : " + prefsFilePath);
+        logger.info("Using preference file : " + prefsFilePath);
 
         UserPrefs initializedPrefs;
         try {
             Optional<UserPrefs> prefsOptional = storage.readUserPrefs();
+            if (!prefsOptional.isPresent()) {
+                logger.info("Preference file " + prefsFilePath
+                        + " not found. Will be starting with a new default preference file.");
```

Comment:
The 'not found' part still feels like an error. I'm wondering if we should just do something like this instead?
```suggestion
                logger.info("Creating a new new preference file " + prefsFilePath);
```

Reply:
I think it could be the case where the file is missing/deleted where the other files exist. I think it really depends on if whether the previous files already exist. 

e.g. 
if config exist, but not preference and data before launch
or if config, preference exist but not data before launch

Should we just standardize all of this to "creating a new XXX file..." or make it a separate case should certain files already exist prior to this.

Another problem is when the file does not have read access, the IOException is caught in the read function so the program will think that the file does not exist (even though it is due to a lack of permission to read). I think this should be solved in a separate PR since it is a separate issue.

Reply:
Good point @Eclipse-Dominator. 
There is also the possibility that all three files are deleted, in which case we can't differentiate between new vs deleted. So, I guess we can simply standardize to "creating a new XXX file..." 

>Another problem is when the file does not have read access, the IOException is caught in the read function so the program will think that the file does not exist (even though it is due to a lack of permission to read). I think this should be solved in a separate PR since it is a separate issue.

Yes, we can address that separately.

Reply:
I have updated the logs message and you can refer to them in the screenshot in the PR.

Reply:
Looks good @Eclipse-Dominator 
You can go ahead and submit the next iteration.